cmake/projects/SuiteSparse/hunter.cmake

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2017, NeroBurner
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cacheable)
+
+hunter_cacheable(SuiteSparse)
+
+# List of versions here...
+## suitesparse-metis-for-windows version is 1.3.1
+## suitesparse library version is 4.5.1
+hunter_add_version(
+    PACKAGE_NAME
+    SuiteSparse
+    VERSION
+    "4.5.1-p0"
+    URL
+    "https://github.com/NeroBurner/suitesparse-metis-for-windows/archive/hunter_v1.3.1.zip"
```

Comment:
You can use this repository:
* https://github.com/hunter-packages/suitesparse-metis-for-windows